mkdocs/commands/gh_deploy.py

```diff
@@ -126,9 +127,15 @@ def gh_deploy(config, message=None, force=False, ignore_version=False):
         if host is None:
             # This could be a GitHub Enterprise deployment.
             log.info('Your documentation should be available shortly.')
+        if page_type == 'user' or page_type == "org":
+            username, repo = path.split('/', 1)
+            if repo.endswith('.git'):
+                repo = repo[:-len('.git')]
+            url = 'https://{}.github.io/'.format(username)
+            log.info('Your documentation should shortly be available at: ' + url)
         else:
             username, repo = path.split('/', 1)
             if repo.endswith('.git'):
                 repo = repo[:-len('.git')]
             url = 'https://{}.github.io/{}/'.format(username, repo)
-            log.info('Your documentation should shortly be available at: ' + url)
+            log.info('Your documentation should shortly be available at: ' + url)
```

Comment:
For some reason the newline at the end of this file was removed, which is resulting in a flake8 error.